test/display_test.py

```diff
@@ -65,20 +65,53 @@ def todo_test_flip(self):
 
         self.fail()
 
-    def todo_test_get_active(self):
+    def test_get_active(self):
+        """Test the get_active function"""
+
+        #Initially, the display is not active
+        pygame.quit()
```

Comment:
don't need this pygame.quit() here. The display tests only init the display module before each test with the setUp() function:

https://github.com/pygame/pygame/blob/4940b9c664f8b063e8f634857c9f79ec5eb736d7/test/display_test.py#L18-L19